### ISSUE TITLE
Explore allowing Producer Certification search on Products page

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/sib-discovery-components#49]
The thing about not having certifications filter on the product page is glitching me a bit. Situation summary:

- We can't / don't want to take on connecting certifications to specific products so we don't have this
- It is likely that it will be a blocker for buyers, if they're looking for products it's an important thing to know
- Do the actual certificates say the actual products on them?

Could we:
* add the Certifications filter onto Products page, having it work on Producer, as per the Producers page 
* change the label / title so it is explicitly "Producer Certifications" AND have a notice / warning that buyer needs to check the certification itself to confirm which products are covered (where / how might we do that?)

**Comment by kirstenalarsen** [comment]
@mkllnk  @JbPasquier am I right in thinking it would be pretty straight forward to have this Certifications filter on the Products page, and to change the title?

**Comment by Louiscsn** [comment]
Hello @kirstenalarsen 
Just to understand: is this a request for evolution?
Should this be part of a new contract between OFN and Startin'blox? :-)

**Comment by kirstenalarsen** [comment]
Yes, accept that this is next scope. Would be good to have an estimate though in case we can get it done for launch

**Comment by mariocarabotta** [comment]
requires buyers research
